src/AzureClient/AzureClient.cs

```diff
@@ -56,6 +57,17 @@ private string ValidExecutionTargetsDisplayText
             EntryPointGenerator = entryPointGenerator;
             Logger = logger;
             eventService?.TriggerServiceInitialized<IAzureClient>(this);
+
+            if (engine is BaseEngine baseEngine)
+            {
+                baseEngine.RegisterDisplayEncoder(new CloudJobToHtmlEncoder());
+                baseEngine.RegisterDisplayEncoder(new CloudJobToTextEncoder());
+                baseEngine.RegisterDisplayEncoder(new CloudJobToJsonEncoder());
```

Comment:
Early comment, but for JSON, you can also [directly register the JsonConverter class](https://github.com/microsoft/jupyter-core/blob/master/examples/moon-kernel/MoonEngine.cs#L31).

Reply:
Thanks! That's much cleaner, just made that change.